tests/test_config/1-star.yaml

```diff
@@ -0,0 +1,19 @@
+# For testing purpose
```

Comment:
Is this file used? Is anything else in this test_config directory?

Reply:
save as above.